nengo_loihi/tests/test_neurons.py

```diff
@@ -83,3 +83,114 @@ def test_loihi_neurons(neuron_type, Simulator, plt, allclose):
     atol = 1. / t_final  # the fundamental unit for our rates
     assert allclose(nengo_rates, ref, atol=atol, rtol=0, xtol=0)
     assert allclose(loihi_rates, ref, atol=atol, rtol=0, xtol=0)
+
+
+@pytest.mark.parametrize(
+    'neuron_type', [LoihiLIF(amplitude=1.0, tau_rc=0.02, tau_ref=0.002),
+                    LoihiLIF(amplitude=0.063, tau_rc=0.05, tau_ref=0.001)])
+def test_nengo_dl_loihi_lif(neuron_type, plt, allclose):
+    pytest.importorskip('nengo_dl')
+    import nengo_dl
```

Comment:
`importorskip` actually returns the imported module, so you can do `nengo_dl = pytest.importorskip("nengo_dl")`